### ISSUE TITLE
Pathogenicity filter: clarification on intended behavior

### ISSUE THREAD

**Issue by nicklecompteBCH** [macarthur-lab/seqr#1227]
**Is your feature request related to a problem? Please describe.**
When filtering variants on "Not Benign," I had thought that this would mean "anything without an explicit ClinVar Benin/Likely Benign annotation." 

However, based on observing our installation of Seqr and looking at the code (specifically ui/panel/search/constants.js), it seems that the behavior is "only include those with an HGMD annotation or a Path/LikelyPath/VUS ClinVar annotation." So many variants without either annotation will be filtered out by "Not Benign." This seems counterintuitive to me, and we have a user here (at Boston Children's Hospital) who has used the Broad's Seqr instance and said he thought the pathogenicity filter worked differently.

That said: we have a forked repo (for Python 3 and some BCH-specific needs) and have used Hail 0.2 for the pipeline to Elasticsearch...so there could well be other problems.

**Describe the solution you'd like**
Could you confirm the expected behavior? FWIW, I think having at least the option to include/exclude variants without a pathogenicity annotation would be a nice-to-have.

**Comment by hanars** [comment]
"Not Benign" is a shortcut for "Variants that are known to be not benign" which means they need to be annotated win clinvar or hgmd as something that is not benign, and that filter has no impact on variants that are absent form clinvar/hgmd. 

Pathogenicity filtering works in parallel with annotation filter, so if you specify any of the pathogenicity options and any of the annotation options it will return results that match either. So if you want variants that are absent from clinvar/hgmd I recommend you use the annotations filter instead, or simply specify neither filter.
